source/content/shibboleth-sso.md

```diff
@@ -215,9 +215,9 @@ Non-Composer implementations must add the following lines to the `settings.php`
 $conf['simplesamlphp_auth_installdir'] = $_ENV['HOME'] .'/code/private/simplesamlphp';
 ```
 
-### Drupal 10
+### Drupal (Latest)
 
-Drupal 10 includes Symfony 6. The SimpleSAMLphp library is not currently compatible with Symfony 6 unless you use the dev-master branch. There is a new `simplesamlphp_auth branch (4.x)` that you can use if you require a Drupal 10 compatibility workaround. Note that this workaround requires you to use dev versions at your own risk.
+The latest version of Drupal includes Symfony 6. The SimpleSAMLphp library is not currently compatible with Symfony 6 unless you use the dev-master branch. There is a new `simplesamlphp_auth branch (4.x)` that you can use if you require a compatibility workaround. Note that this workaround requires you to use dev versions at your own risk.
```

Comment:
Yes, Drupal 10 includes Symfony 6. When the latest version of Drupal is Drupal 11, then it will probably use some later version of Symfony. I'd therefore keep this "Drupal 10 includes Symfony 6".

Reply:
Agree that this change should be reverted.. but I find the language here concerning more generally in terms of our (in)ability to keep the documentation up to date with the latest status of interoperability of various community projects and versions.  I'm not sure what the right answer is here (or if there even is one)... could we link out to some external source of truth for this?